docs/monthly-meeting/2024-07.md

```diff
@@ -0,0 +1,120 @@
+- Jim DeLaHunt / `@JDLH`
+- Ryan / `ryan-duve`
+- Hugo van Kemenade / `@hugovk`
+- Trey
+
+## Introductions
+
+> If there are any new people, we should do a round of introductions.
+
+
+
+## Reports and celebrations
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- [Hugo] Plausible stats are live!
+  [plausible.io/docs.python.org](https://plausible.io/docs.python.org) &
+  [plausible.io/packaging.python.org](https://plausible.io/packaging.python.org)
+
+- [Hugo] Sphinx meetup tomorrow at 20 UTC:
+  [sphinx-doc/discussions/12443](https://github.com/orgs/sphinx-doc/discussions/12443#discussioncomment-9936592)
+
+## Discussion
+
+- [Daniele] Options for hosting a Python documentation sprint
+  (Cardiff or London, possibly Netherlands)
+  - Daniele ran a successful docs sprint @ DjangoCon.
+    - People choose anything that bothers you in the docs, determine the rule it
+      breaks, choose what you'd do right now to fix it. Daniele can "bless" the
+      idea, or not; if blessed the person can work on it.
+    - Some commits made it into the docs
+  - Daniele can arrange a sprint in one of the 3 locations
+  - [Ned] A sprint doesn't have to be a big formal thing; it can be a meetup
+  - [Daniele] Some formal structure helped
+  - [Ned] You don't need special permission
+  - [Daniele] Would a core dev be interested in joining?
+    - Possibly: Alex, Pradyun, Tania Allard, Petr
+  - [Hugo] Last year Daniele led the workshop & Hugo was clicking the merge
+    button at the sprint, worked very well
+  - [Petr] worried about the conference helping to get people to the event
+  - [Carol] The [packaging guide (Python Packaging User Guide)](https://packaging.python.org/en/latest/)
+    could also use some work
+  - [Ned] interested in the idea spreadsheet; maybe it can help even offline
+  - [Carol] At last PyCon US, Shauna and Melanie tried to make their first docs
+    PR, and found some issues in the onboarding process
+    - Shauna: https://docs.google.com/document/d/1icjPpmEUH0BRPDwl7oXLkpasvwZoS5io2LyTEhblku0/edit
+    - Melanie: https://docs.google.com/document/d/11zxisx5XfYOsrDOn4qd-XUSpDltIOi35qiOO9fDLfnI/edit
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, etc.*
+
+- [Ryan] I need to load JSON with NaN values.
+  [The official documentation](https://docs.python.org/3/library/json.html#json.load) says:
+  ***parse_constant**, if specified, will be called with one of the following strings:
+  '-Infinity', 'Infinity', 'NaN'. This can be used to raise an exception if
+  invalid JSON numbers are encountered.*
```

Comment:
```suggestion
  [The official documentation](https://docs.python.org/3/library/json.html#json.load) says:
  > **parse_constant**, if specified, will be called with one of the following strings:
  > `'-Infinity'`, `'Infinity'`, `'NaN'`. This can be used to raise an exception if
  invalid JSON numbers are encountered.
```